databricks/koalas/frame.py

```diff
@@ -6684,7 +6684,7 @@ def _reindex_columns(self, columns):
 
         return self._internal.copy(sdf=sdf, data_columns=columns, column_index=idx)
 
-    def melt(self, id_vars=None, value_vars=None, var_name='variable',
+    def melt(self, id_vars=None, value_vars=None, var_name=None,
```

Comment:
Seems the default value of `melt`'s `var_name` at namespace.py should be changed as well.

Reply:
ah, good catch!